src/diffusers/pipelines/pipeline_utils.py

```diff
@@ -672,6 +678,7 @@ def from_pretrained(cls, pretrained_model_name_or_path: Optional[Union[str, os.P
         offload_state_dict = kwargs.pop("offload_state_dict", False)
         low_cpu_mem_usage = kwargs.pop("low_cpu_mem_usage", _LOW_CPU_MEM_USAGE_DEFAULT)
         variant = kwargs.pop("variant", None)
+        dduf_file = kwargs.pop("dduf_file", None)
```

Comment:
This needs to be also version-guarded with a `transformers` version, right?

Reply:
or update the lower-bound in `setup.py` ?

Reply:
That works too.

Reply:
I added a version guard a bit later in the pipeline. Will this be enough or you prefer to raise an error earlier ?

Reply:
I prefer not updating the lower-bound of transformers if possible.

Reply:
I think we should raise errors as early as possible.